torch/csrc/jit/script/compiler.cpp

```diff
@@ -1396,17 +1397,69 @@ struct to_ir {
       const Ident& target,
       const List<Expr>& args,
       const List<Stmt>& body) {
-    // TODO: start, stop, step loop
-    if (args.size() != 1) {
+    Value *end_val, *start_val, *step_val;
```

Comment:
It's probably safer to initialize these as `nullptr` and assert they are non-null in control flow blocks below when they're about to be used. This will help others who modify this code in the future